docs/InstallGuide.md

```diff
@@ -131,294 +138,221 @@ To automatically launch the mysql server, execute the following command.
 
```

Comment:
"To automatically launch the mysql server" doesn't accurately explain what this command does. It should be: "To automatically launch the mysql server at boot"